src/services/RequestService.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (c) 2021,2022 BMW Group AG
```

Comment:
Did you remove this because it was wrong, because of some c&p error or is this a misstake?